### PR TITLE
New version: HHDSoftware.FreeHexEditorNeo version 7.50.02.8795

### PR DESCRIPTION
**Pull request by PckgrMiranda**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/193185)

### CODE REVIEW

**Comment by PckgrMiranda** [comment]
@PckgrTom

File: manifests/h/HHDSoftware/FreeHexEditorNeo/7.50.02.8795/HHDSoftware.FreeHexEditorNeo.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+PrivacyUrl: https://www.hhdsoftware.com/company/privacy-policy
+Author: HHD Software, Ltd.
+PackageName: HHD Software Free Hex Editor Neo
+PackageUrl: https://www.hhdsoftware.com/free-hex-editor
+License: Freeware
+LicenseUrl: https://www.hhdsoftware.com/Official/software-licenses/HEXFREE
+Copyright: Copyright (c) HHD Software Ltd. All rights reserved.
+CopyrightUrl: https://www.hhdsoftware.com/Official/software-licenses/HEXFREE
+ShortDescription: Free:Freeware binary files editing tool
+Description: Free:Freeware binary files editing tool
```
Comment (Trenly) [inline manifests/h/HHDSoftware/FreeHexEditorNeo/7.50.02.8795/HHDSoftware.FreeHexEditorNeo.locale.en-US.yaml:19]:
```suggestion
Description: Free Freeware binary files editing tool
```

File: manifests/h/HHDSoftware/FreeHexEditorNeo/7.50.02.8795/HHDSoftware.FreeHexEditorNeo.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+PublisherSupportUrl: https://www.hhdsoftware.com/support/contact-us
+PrivacyUrl: https://www.hhdsoftware.com/company/privacy-policy
+Author: HHD Software, Ltd.
+PackageName: HHD Software Free Hex Editor Neo
+PackageUrl: https://www.hhdsoftware.com/free-hex-editor
+License: Freeware
+LicenseUrl: https://www.hhdsoftware.com/Official/software-licenses/HEXFREE
+Copyright: Copyright (c) HHD Software Ltd. All rights reserved.
+CopyrightUrl: https://www.hhdsoftware.com/Official/software-licenses/HEXFREE
+ShortDescription: Free:Freeware binary files editing tool
```
Comment (Trenly) [inline manifests/h/HHDSoftware/FreeHexEditorNeo/7.50.02.8795/HHDSoftware.FreeHexEditorNeo.locale.en-US.yaml:18]:
```suggestion
ShortDescription: Free Freeware binary files editing tool
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/h/HHDSoftware/FreeHexEditorNeo/7.50.02.8795/HHDSoftware.FreeHexEditorNeo.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  Silent: -silent
+  SilentWithProgress: -silent
+  Custom: -user
+UpgradeBehavior: install
+ReleaseDate: 2024-11-21
+Installers:
+- Architecture: x64
+  InstallerUrl: https://hhdsoftware.com/download/free-hex-editor-neo.exe
+  InstallerSha256: B8ACBDFF21CE28CDED12A4F3F5F829FB227AA07365817F7E845F08716DC017DE
+  InstallerSwitches: {}
```
Comment (Trenly) [inline manifests/h/HHDSoftware/FreeHexEditorNeo/7.50.02.8795/HHDSoftware.FreeHexEditorNeo.installer.yaml:25]: @russelbanks - Is this a Komac error?

https://github.com/PckgrMiranda/winget-pkgs/blob/HHDSoftware.FreeHexEditorNeo-7.50.02.8795-99762F06F864461389F1E452320322EF/manifests/h/HHDSoftware/FreeHexEditorNeo/6.44.01.6234/HHDSoftware.FreeHexEditorNeo.installer.yaml

Comment (Trenly) [reply manifests/h/HHDSoftware/FreeHexEditorNeo/7.50.02.8795/HHDSoftware.FreeHexEditorNeo.installer.yaml:25]: @PckgrMiranda - Not sure why this happened, but for some reason the switches didn't copy over from the previous version

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**